src/main/java/edu/kis/powp/jobs2d/command/loader/LoadCommand.java

```diff
@@ -0,0 +1,69 @@
+package edu.kis.powp.jobs2d.command.loader;
+
+import edu.kis.powp.jobs2d.command.CompoundCommand;
+import edu.kis.powp.jobs2d.command.DriverCommand;
+import edu.kis.powp.jobs2d.command.OperateToCommand;
+import edu.kis.powp.jobs2d.command.SetPositionCommand;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class LoadCommand {
+
+    public static CompoundCommand loadCommandsFromFile(String fileName) {
+        List<String> lines = readLinesFromFile(fileName);
+        List<DriverCommand> commands = new ArrayList<>();
+
+        for (String line : lines) {
+            commands.add(getCommandFromLine(line));
```

Comment:
Format of import fixed. No OCP: adding new format (e.g., json) cannot reuse this code without modification.

Reply:
Done